src/components/Dropdown/styles.scss

```diff
@@ -0,0 +1,82 @@
+.container{
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+}
+
+.dropdown{
+    all: unset;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 56px;
+    padding: 16px;
+    margin-bottom:8px;
+    background: #F5F5F5;
+    border-radius: 8px;
+    box-sizing: border-box;
+    &:active{
+        background: #BBBBBB;
```

Comment:
I'm not sure if you have a theme file, but try to declare colors as theme variables